.travis.yml

```diff
@@ -9,7 +9,11 @@ jobs:
       os: linux
       dist: xenial
       language: python
-      python: 2.7.14
+      python:
+        - 3.5
+        - 3.6
```

Comment:
1) no need to run flake on all pythons, oldest will be enougth.  It just a spell checker.
2) list of pythons will not work as you expect, only first one will be used.